recipes/meta.yaml

```diff
@@ -0,0 +1,53 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "vifm" %}
+{% set version = "0.8.2" %}
+{% set sha256 = "8b466d766658a24d07fc2039a26fefc6a018f5653684a6035183ca79f02c211f" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://jaist.dl.sourceforge.net/project/vifm/vifm/{{ name }}-{{ version }}.tar.bz2
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - ncurses
+
+  run:
+    - libgcc
```

Comment:
If you don't need `gcc` library at build time you probably don't need `libgcc` at run time.